src/main/java/com/google/sps/utility/AuthenticationUtility.java

```diff
@@ -40,6 +40,13 @@
   // Application Name
   public static final String APPLICATION_NAME = "PUT NAME HERE";
 
+  // Get JsonFactory used for building Google API Service Instances
+  public static final JsonFactory JSON_FACTORY = new JacksonFactory();
+
+  // Get HTTPTransport appropriate for App Engine environment
+  // https://googleapis.dev/java/google-http-client/latest/com/google/api/client/extensions/appengine/http/UrlFetchTransport.html
+  public static final HttpTransport HTTP_TRANSPORT = new UrlFetchTransport();
```

Comment:
Nit: Just noticed that they both offer `getDefaultInstance()`. I think that is preferred over creating your own instances. If you follow that route, then I think it makes more sense for you to simply call `getDefaultInstance()` where needed instead of defining these constants.

Reply:
Huh interesting - thanks for pointing this out!